tests/inventory

```diff
@@ -0,0 +1,2 @@
+[local]
```

Comment:
There is already an `ci/` folder for testing, why adding this one or not removing the other ?